io/runes/runes.go

```diff
@@ -0,0 +1,89 @@
+package runes
+
+import (
+	"fmt"
+	"io"
+	"unicode"
+)
+
+// Reader is the interface that wraps the basic Read method.
+//
+// Read reads up to len(data) runes into data. It returns the number of runes
```

Comment:
I decided to copy/adjust the `io.Reader` documentation here instead of redirecting the user somewhere else because it'd be confusing to read the documentation about reading bytes if this is a rune reader.